components/woo/order-bulk.php

```diff
@@ -430,6 +430,28 @@ protected function create_label_for_order( $order_id, $request ) {
 	}
 
 	/**
+	 * Sanitize package data.
+	 *
+	 * User enter package data that can:
+	 *
+	 * - Have local decimal separator.
+	 *
+	 * @since 1.5.1
+	 *
+	 * @param array $package_data
+	 *
+	 * @return array
+	 */
+	protected function sanitize_package( $package_data ) {
+		$package_data['width']  = wc_format_decimal( $package_data['width'] );
```

Comment:
I'm getting the following error while trying to create a bulk shipment:
```
Uncaught Error: Cannot use object of type ShipcloudDomainPackage as array in /kunden/440816_22081/webseiten/woocommerce-shipcloud-dev/wordpress/wp-content/plugins/shipcloud-for-woocommerce/components/woo/order-bulk.php:446
```

Reply:
Interesting. Thanks to that I found a complete different approach.